config/nginx.conf.erb

```diff
@@ -268,6 +268,18 @@ http {
                   return 302 https://www.python.org/dev/peps/;
                 }
 
```

Comment:
The redirects above could have been updated too

Reply:
Just getting the minimum in place before we transition to permanent redirects. Once this has been out in the wild for a bit, I'll simplify the whole setup.

Also plan to add a Surrogate-Key so we can purge all the redirects being dispatched from Nginx.